run_tests.sh

```diff
@@ -1,12 +1,12 @@
 #!/bin/sh
 # Postgresql and ganache-cli must be running for the tests
 
-ps aux | grep ganache-cli | grep -v grep > /dev/null
+ps aux | grep ganache | grep -v grep > /dev/null
 if [ $? -eq 1 ]; then
     echo 'Running Ganache-Cli'
-    ganache-cli --defaultBalanceEther 10000 --gasLimit 10000000 -a 30 --chain.chainId 1337 --chain.networkId 1337 -d > /dev/null &
+    npx ganache@7.9.1 --defaultBalanceEther 10000 --gasLimit 10000000 -a 30 --chain.chainId 1337 --chain.networkId 1337 -d > /dev/null &
```

Comment:
Could you set latest instead of pinning an address? That's what we do in the CI and I don't want to go manually checking this script

Reply:
I think setting the latest is prone to errors on major releases. If some CLI flag is not recognized, it will not error and continue executing, and it's hard to catch. What do you think about putting the latest minor release?

The motivation for pinning a version is that I ran into errors because I was using an older version, and there needed to be info about what version I should use.

Reply:
For example, if you have ganache v6 installed, the tests will run but fail, and it took me a while to realise what was wrong,

Reply:
> I think setting the latest is prone to errors on major releases. If some CLI flag is not recognized, it will not error and continue executing, and it's hard to catch. What do you think about putting the latest minor release?

I think then we will have to inform the user ganache is not executing and don't launch the tests. I want errors related to RPC to be detected as soon as we can, and using the latest ganache version for us is mandatory. I don't want to be checking this script from time to time. Also maybe we could remove `npx` dependency here and add `ganache` to `docker-compose`

Reply:
> lso maybe we could remove npx dependency here and add ganache to docker-compose

Technically npx is a transitive dependency of ganache because it has to be installed with npm, and npx is bundled with npm, so it's fine. I do like the idea of adding ganache to the docker image as it seems to be more aligned with how the CI works.

Reply:
Updated boss @Uxio0